web/data/help.mdx

```diff
@@ -179,6 +180,9 @@ If you experience crashes or severe glitches when using savestates, your platfor
 |                               | - Phan Flame Pattern: Choose one of four patterns for Phantoon's flames
 |                               | - Next Flame Pattern: Swaps with Phan Flame Pattern each time Phantoon chooses a pattern
 | - Botwoon RNG                 | Choose the opening pattern for Botwoon's first cycle
+| - Botwoon 2nd+                | Choose the movement pattern for Botwoon's next cycle
```

Comment:
Was renamed to Botwoon Second. Maybe write some more notes about the pattern naming.